CHANGELOG.md

```diff
@@ -0,0 +1,9 @@
+# Changelog for IncidentIo v0.1.0
+
+The format is based on [Keep a Changelog](https://keepachangelog.com/en/1.0.0/).
+
+## [Unreleased]
+
+## [0.1.0] - 2024-07-03
```

Comment:
This should link to the tag for v0.1.0. Perhaps apply it as part of the release itself?
```suggestion
## [0.1.0](https://github.com/sgerrand/ex_incident_io/tag/v0.1.0) - 2024-07-03
```